ci-operator/step-registry/openshift/image-based/upgrade/e2e/conf/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
```

Comment:
I saw in some other folders we symlink to the parent folder, should we do that as well?

Reply:
I've been thinking of converting all of the ibu/ibi stuff to that approach, I'll open a separate PR with that